# _LIBADD is for object files, not libraries, except in the top level,
# it's how we build the framework.  So move the Kerberos libraries to
# the top level.
#
# The Kerberos framework on Mac OS X defines error_message, and the
# pidgin version of Zephyr code does too.  Since we don't care about
# actually invoking it, rename the local symbol.
#
# 
#
# old_revision [20e2add42a314b4914e798757eb2a69ac3b491d0]
#
# patch "libpurple/Makefile.am"
#  from [fb4d8727ecc906ce6a12693a0e4486da802a9997]
#    to [de6fb0ff2bee79eb4359a81b3aca4be9ba685bb2]
# 
# patch "libpurple/protocols/zephyr/Makefile.am"
#  from [c3c7b401feed6ef0512e6db95aa8e001d3e0c7be]
#    to [b474843d4a3dcca1ebfb36a1df3be83568a3843c]
#
============================================================
--- libpurple/Makefile.am	fb4d8727ecc906ce6a12693a0e4486da802a9997
+++ libpurple/Makefile.am	de6fb0ff2bee79eb4359a81b3aca4be9ba685bb2
@@ -232,6 +232,7 @@ libpurple_la_LIBADD = \
 libpurple_la_LDFLAGS = -export-dynamic -version-info $(PURPLE_LT_VERSION_INFO) -no-undefined
 libpurple_la_LIBADD = \
 	$(STATIC_LINK_LIBS) \
+	$(KRB4_LDFLAGS) $(KRB4_LIBS) \
 	$(DBUS_LIBS) \
 	$(GLIB_LIBS) \
 	$(LIBXML_LIBS) \
============================================================
--- libpurple/protocols/zephyr/Makefile.am	c3c7b401feed6ef0512e6db95aa8e001d3e0c7be
+++ libpurple/protocols/zephyr/Makefile.am	b474843d4a3dcca1ebfb36a1df3be83568a3843c
@@ -80,8 +80,7 @@ libzephyr_a_SOURCES = $(ZEPHYRSOURCES)
 st = -DPURPLE_STATIC_PRPL -Dlint
 noinst_LIBRARIES = libzephyr.a
 libzephyr_a_SOURCES = $(ZEPHYRSOURCES)
-libzephyr_a_CFLAGS  = $(AM_CFLAGS)
-libzephyr_a_LIBADD  = $(ZEPHYRLIBS)
+libzephyr_a_CFLAGS  = $(AM_CFLAGS) -Derror_message=z_error_message
 
 else
 
